statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -1536,9 +1536,11 @@ def fill_coll(sim):
             ma_coll[i, :, :, :] = fill_coll(sim)
 
         ma_sort = np.sort(ma_coll, axis=0)  # sort to get quantiles
-        index = round(signif/2*repl)-1, round((1-signif/2)*repl)-1
-        lower = ma_sort[index[0], :, :, :]
-        upper = ma_sort[index[1], :, :, :]
+        # python 2: round returns float
+        low_idx = int(round(signif / 2 * repl) - 1)
```

Comment:
Consider copy/pasting smoke tests from #3638.

Reply:
I added a test in my recent VAR/exog PR that is more than a smoke test and that triggered the compat failure on Travis in this PR

Reply:
Great.  Just wanted to make sure you knew its available.  I'm very happy to see this finally fixed.